src/cpp/shared/game/game.h

```diff
@@ -153,7 +153,7 @@ class game {
 	char numObelisks;
 	town castles[MAX_TOWNS];
 	char field_2773[72];
-	char field_27BB[9];
+	char builtToday[9];
```

Comment:
Eliminate this magic number!

This number is: 

ROUNDUP(MAX_CASTLES, 8)


You can use this definition for ROUNDUP

#define ROUNDUP(a, n)                                           \
({                                                              \
        uint32_t __n = (uint32_t) (n);                          \
	(typeof(a)) (ROUNDDOWN((uint32_t) (a) + __n - 1, __n)); \
})

Reply:
What's the reason it's rounded up though?

Reply:
Suppose there were max 73 castles instead of max 72 castles. Then this field would have been 10 bytes. 73 / 8 is 9, but ROUNDUP(73, 8) is 10.

Reply:
Oh, I forgot the definition of ROUNDDOWN:

#define ROUNDDOWN(a, n)                                         \
({                                                              \
        uint32_t __a = (uint32_t) (a);                          \
        (typeof(a)) (__a - __a % (n));                          \
})

Reply:
Better formatted definitions: https://gist.github.com/jkoppel/00ccd01ee22f5c89fe89c5b32819dd71

Reply:
I understand how these functions work, but do you know why the size of these is like that?